iceoryx_binding_c/test/moduletests/test_listener.cpp

```diff
@@ -50,6 +51,7 @@ using namespace ::testing;
 iox_user_trigger_t g_userTriggerCallbackArgument = nullptr;
 iox_sub_t g_subscriberCallbackArgument = nullptr;
 iox_client_t g_clientCallbackArgument = nullptr;
+iox_service_discovery_t g_serviceDiscoveryCallbackArgument = nullptr;
```

Comment:
If we use the global variable in multiple tests we need to reset it in the corresponding setups at least. I am not sure we use it in multiple tests though.